### ISSUE TITLE
Atomic typo (FR)

### ISSUE THREAD

**Issue by SasquatchYuja** [signalapp/Signal-Android#9603]
- [X] I have searched open and closed issues for duplicates
- [X] I am submitting a bug report for existing functionality that does not work as intended
- [X] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [X] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Literally atomic typo.
Not sure whether it belongs here though, I've seen issues and PRs redirected to [Transifex](https://www.transifex.com/signalapp/signal-android/) but also some that have been merged. I am not familiar with Transifex but I assume there is a difference between actual translation and misspelling, whose fixup makes no sense to be translated, no ?

### Steps to reproduce
- enter a chat
- 3-dots menu, top right corner
- All media (FR: "Tous les médias")
- sort option: Newest (FR: "Le pPlus récent") --> should be "Le plus récent"

### Device info
**Device:** Samsung SM-G930F
**Android version:** 8.0.0
**Signal version:** 4.59.8 (last)

[0001-Typo-fixed-French-in-media-sorting-naming.patch.txt](https://github.com/signalapp/Signal-Android/files/4598666/0001-Typo-fixed-French-in-media-sorting-naming.patch.txt)

**Comment by alan-signal** [comment]
Thank you but translations are done only on [transifex.com](https://www.transifex.com/signalapp/signal-android/dashboard/). If we merged them here they would only be overwritten.

We're always looking for volunteers who can [help us keep our translations up-to-date](https://www.transifex.com/signalapp/signal-android/dashboard/). We appreciate your help.

**Comment by alan-signal** [comment]
I'm going to reopen as this is an issue still, but just note we cannot accept the patch.

**Comment by SasquatchYuja** [comment]
That's why I didn't make a PR, just joined the patch so that it is easy to find out where the issue is located without having to grep the repo. Maybe there's an other way in Transifex to locate strings relating to their appearance in the app. Will try to remember creating an account there. If I do I will make the change.

> Can you please link to them

Looking closely I talked too fast, there are indeed example of PR ending up being rebased but it concerns typos in code files, like #8895

To sum up typos in code file (whether they [break](https://github.com/signalapp/Signal-Android/pull/5723) or not) still belong here, whereas those in /app/src/main/res/values-[a-z] are to be fixed in Transifex.
I believed typos were treated differently than real translations, sorry for misunderstanding.

**Comment by alan-signal** [comment]
Don't worry, thanks for the report.

> Maybe there's an other way in Transifex to locate strings relating to their appearance in the app. Will try to remember creating an account there. If I do I will make the change.

Yes, you can search by English:

https://www.transifex.com/signalapp/signal-android/translate/#fr/master/186104089?q=text%3ANewest or translated text:

https://www.transifex.com/signalapp/signal-android/translate/#fr/master/186104089?q=translation_text%3ApPlus

![image](https://user-images.githubusercontent.com/48254818/81449004-b7caf280-9155-11ea-8b1a-332615e1e981.png)

I'll leave it over the weekend to give you a chance to fix if you like. I'll do it myself Monday if I see it's still there.

**Comment by SasquatchYuja** [comment]
Join request to the French team still pending. Don't bother waiting, I will know how to proceed next time I notice a mistake

**Comment by Meteor0id** [comment]
@alan-signal even when he gets accepted he can't change that string, it has been marked reviewed. I pinged the guy who translated and reviewed this string so hopefully he will come and set it right. He is usually quick to respond.
